acra-censor/acra-censor_configuration_provider.go

```diff
@@ -107,7 +108,7 @@ func (acraCensor *AcraCensor) LoadConfiguration(configuration []byte) error {
 			go queryCaptureHandler.Start()
 			acraCensor.AddHandler(queryCaptureHandler)
 		default:
-			acraCensor.logger.Errorln("Unexpected handler in configuration")
+			acraCensor.logger.WithField(logging.FieldKeyEventCode, logging.EventCodeErrorCensorSetupError).Errorln("Unexpected handler in configuration")
```

Comment:
I find long lines like this a bit hard to read. Something like this would be clearer, I think.
```suggestion
			acraCensor.logger.
				WithField(logging.FieldKeyEventCode, logging.EventCodeErrorCensorSetupError).
				Errorln("Unexpected handler in configuration")
```

Though I don't know if one can convince **gofmt** that this code style is okay.